.eslintrc.js

```diff
@@ -1,14 +1,24 @@
 module.exports = {
-    "env": {
-        "browser": true,
-        "commonjs": true,
-        "es2020": true,
-        "node": true,
+    parser: 'babel-eslint',
+    env: {
+        browser: true,
+        commonjs: true,
+        es2020: true,
+        node: true,
     },
-    "extends": "eslint:recommended",
-    "parserOptions": {
-        "ecmaVersion": 12
+    extends: ['@10up/eslint-config', '@10up/eslint-config/wordpress'],
+    parserOptions: {
+        ecmaVersion: 12,
+        ecmaFeatures: {
+            jsx: true,
+        },
+    },
+    rules: {},
+    globals: {
+        module: true,
+        process: true,
+        window: true,
+        document: true,
+        wp: true,
```

Comment:
This .eslintrc.js is only for this repo itself.

Consumers write config files in postinstall.js. It's possible it now has to be refactored to reading files instead of hardcoded string literals.

Many tools are configured per-repo, because with ESLint and similar there are no universal settings, or they're ultra-minimal. Some plugin might require jsx, theme might not.

Reply:
> This .eslintrc.js is only for this repo itself.
> 
> Consumers write config files in postinstall.js. It's possible it now has to be refactored to reading files instead of hardcoded string literals.
> 
> Many tools are configured per-repo, because with ESLint and similar there are no universal settings, or they're ultra-minimal. Some plugin might require jsx, theme might not.

I am importing the `lint-staged-config` config files, into the project and then override it if needed. So shouldn't be an issue to keep the defaults.

Reply:
> So shouldn't be an issue to keep the defaults.

Not convinced (yet). I agree that perhaps different config "starter" templates should be provided in `/eslint/` sub-directory, maybe start with "wordpress-plugin" and "wordpress-theme" templates, see what that would look like.

Separate PR.